special-pages/pages/history/app/components/App.module.css

```diff
@@ -51,13 +55,53 @@ body {
     padding-right: var(--main-padding-right);
     padding-top: 24px;
 }
+
 .customScroller {
     overflow-y: scroll;
-    scrollbar-gutter: stable;
+    padding-right: calc(var(--main-padding-right) - var(--scrollbar-width));
```

Comment:
This helps the alignment of selected rows, ensuring they perfectly match the search bar in the header